chromium_src/chrome/browser/ui/tabs/tab_strip_model.cc

```diff
@@ -85,6 +90,24 @@ void TabStripModel::AppendWebContents(WebContents* contents,
     observer.TabInsertedAt(this, contents, index, foreground);
 }
 
+void TabStripModel::InsertWebContentsAt(int index,
```

Comment:
most of the differences in this class were to handle the active and index being set in browser-laptop, so can we use the chrome version directly now or are there still differences?